circle.yml

```diff
@@ -2,10 +2,17 @@ machine:
   node:
     version: 4.0.0
 test:
-  pre:
+  override:
     - npm run eslint
-    - npm test
-    - nvm install 6.9.1 && nvm use 6.9.1 && npm test
-    - nvm install 7.10.0 && nvm use 7.10.0 && npm test
+    - npm run test-ci
+    - nvm install 6.11.1 && nvm use 6.11.1 && npm run test-ci
+    - nvm install 8.2.1 && nvm use 8.2.1 && npm run test-ci
```

Comment:
I would maybe target the major release; if not the minor but I would not pin the patch 😉